python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1412,17 +1423,20 @@ def get_event_records(
         """
         return self._event_storage.get_event_records(event_records_filter, limit, ascending)
 
+    @public
```

Comment:
It's surprising to me that `get_event_records` is not marked as public.

I know that people are already using `get_event_records`, but I suspect that people are not really using `get_asset_records`.

I think we probably want to move in the direction of `get_asset_records` (and deprecating `get_event_records`), but I don't think that's realistic in terms of how people are using OSS right now.  

cc @alangenfeld @sryza

Reply:
this seems reasonable to me - if we're going to be marking `get_event_records` public, should we also make `RunRecord` available in the top-level API? At the very least, we should probably put it in the API docs.

Reply:
I think that should be safe, yeah